### DISCUSSION TITLE
How can I watch for popups

### DISCUSSION THREAD

**Discussion by oSethoum** [go-rod/rod#924] [Q&A] [answered]
Am using rod to automate some actions in a web app, however there are different kind of popups ( modals, drawers ...etc ) that shows randomly, so I wanna have some kind of watchers for those elements in order to close them as soon as they show up so I can continue the main task. 
How can I do that?

**Comment by ysmood** [comment, marked as answer]
https://github.com/go-rod/rod/blob/0af5471d5b16a38276cdc6a81c2efc39658f8dbe/page_test.go#L648
